addons/events/event-driven-decisions/event-driven-decisions-common/src/main/java/org/kie/kogito/eventdriven/decision/EventDrivenDecisionController.java

```diff
@@ -0,0 +1,177 @@
+/*
+ *  Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.kie.kogito.eventdriven.decision;
+
+import java.net.URI;
+import java.util.Optional;
+import java.util.UUID;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.cloudevents.CloudEvent;
+import org.kie.dmn.api.core.DMNContext;
+import org.kie.dmn.api.core.DMNResult;
+import org.kie.kogito.Application;
+import org.kie.kogito.cloudevents.CloudEventUtils;
+import org.kie.kogito.conf.ConfigBean;
+import org.kie.kogito.decision.DecisionExecutionIdUtils;
+import org.kie.kogito.decision.DecisionModel;
+import org.kie.kogito.decision.DecisionModels;
+import org.kie.kogito.dmn.rest.DMNJSONUtils;
+import org.kie.kogito.event.CloudEventEmitter;
+import org.kie.kogito.event.CloudEventReceiver;
+
+import static org.kie.kogito.eventdriven.decision.DecisionRequestType.EVALUATE_ALL;
+import static org.kie.kogito.eventdriven.decision.DecisionRequestType.EVALUATE_DECISION_SERVICE;
+
+public class EventDrivenDecisionController {
+
+    private static final String VALID_REQUEST_EVENT_TYPE = DecisionRequestEvent.class.getName();
```

Comment:
What about?
```suggestion
    private static final String VALID_REQUEST_EVENT_TYPE = DecisionRequestEvent.class.getSimpleName();
```
I think it is enough without adding the package

Reply:
I used `getName` with the whole package to be consistent with `TraceEvent`s which do the same.
I prefer being consistent over saving the length of the package in this specific event but if you prefer otherwise it's fine for me to change it.

Reply:
New event types use plain string instead of class full names.